vm/interpreter/src/gas_tracker/price_list.rs

```diff
@@ -42,6 +43,52 @@ lazy_static! {
         hashing_base: 31355,
         compute_unsealed_sector_cid_base: 98647,
         verify_seal_base: 2000, // TODO revisit potential removal of this
+
+        verify_aggregate_seal_base: 0,
+        verify_aggregate_seal_per: [
+            (
+                RegisteredSealProof::StackedDRG32GiBV1P1,
+                449900
+            ),
+            (
+                RegisteredSealProof::StackedDRG64GiBV1P1,
+                359272
+            )
+        ].iter().copied().collect(),
```

Comment:
can we not simply do: `vec![ {..} {...}]` ?

Reply:
No this is a AHashMap